sdk/core/azure-core-tracing-opentelemetry/src/main/java/com/azure/core/tracing/opentelemetry/OpenTelemetryTracer.java

```diff
@@ -48,17 +51,30 @@
     @Override
     public Context start(String spanName, Context context) {
         Objects.requireNonNull(spanName, "'spanName' cannot be null.");
-        Objects.requireNonNull(context, "'context' cannot be null.");
+        SpanBuilder spanBuilder = createSpanBuilder(
+            spanName,
+            null,
+            SpanKind.INTERNAL,
+            null,
+            context);
+
+        return startSpanInternal(spanBuilder, null, false, context);
+    }
 
-        SpanBuilder spanBuilder = getSpanBuilder(spanName, context);
-        Span span = spanBuilder.startSpan();
-        if (span.isRecording()) {
-            String tracingNamespace = getOrDefault(context, AZ_TRACING_NAMESPACE_KEY, null, String.class);
-            if (tracingNamespace != null) {
-                span.setAttribute(AZ_NAMESPACE_KEY, tracingNamespace);
-            }
-        }
-        return context.addData(PARENT_SPAN_KEY, span);
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public Context start(StartSpanOptions options, Context context) {
+        Objects.requireNonNull(options, "'options' cannot be null.");
```

Comment:
Do you think we have the flexibility to allow the customers to fall back on the [previosuly defined API](https://github.com/Azure/azure-sdk-for-java/pull/23159/files#diff-a388db0c914f5d982f7222052290d008c37a4b69429d7ab139ec2ad4a9954d1fR52) in case if they want to provide a null options bag. Since `options` suggest it can be optional.

```java 
    public Context start(String spanName, StartSpanOptions options, Context context) {
```

Reply:
sure, makes  sense